### DISCUSSION TITLE
9p fs client over network

### DISCUSSION THREAD

**Discussion by rinor** [nanovms/nanos#1981] [Q&A]
How far are we (code wise) from having Nanos kernel (9pnet klib?) being able to mount 9pfs over network?

**Comment by eyberg** [comment]
we have support via virtfs which uses 9p for host<>guest https://nanovms.com/dev/tutorials/accessing-host-filesystem-from-nanos-guest-using-9p ; prob would be a matter of a use-case (specific app) / time for extending to the network
